examples/todo/examples.text

```diff
@@ -0,0 +1,94 @@
+package script
```

Comment:
It might make it easier for readers to understand this if the example source code we're using to find todos in is *not*, in fact, a program for finding todos! Any other simple program would do, but this is potentially confusing.

Reply:
Will you substitute another program, then? We could use one of the other `script` examples, perhaps.